apps/store/src/components/CartPage/CartPage.tsx

```diff
@@ -13,35 +13,46 @@ export const CartPage = () => {
   const cost = { total: 350, subTotal: 250 }
 
   return (
+    <Wrapper>
     <Space y={3}>
-      <div></div>
-      <PageHeader>
-        <Heading as="h1" variant="standard.24">
-          Cart (2)
-        </Heading>
-      </PageHeader>
+      <Heading as="h1" variant="standard.24">
+        Cart (2)
+      </Heading>
       <CartList products={products} />
       <Footer>
-        <PriceBreakdown currency="SEK" products={products} cost={cost} />
-        <Button fullWidth={true}>
-          <Link href={PageLink.cartReview()}>Check Out</Link>
-        </Button>
+        <Space y={1}>
+          <PriceBreakdown currency="SEK" products={products} cost={cost} />
+          <Button fullWidth={true}>
```

Comment:
This should be sufficient
```suggestion
          <Button fullWidth>
```